src/collections/members/aviral-asthana/index.mdx

```diff
@@ -0,0 +1,15 @@
+---
+name: Aviral Asthana
+position: Contributor
+image_path: ./Aviral-Asthana.webp
+github: Aviral0702
+linkedin: aviral-asthana-02b70824b
+layer5: 3f770688-e487-45c2-943b-c94b950d3d69
+location: Ghaziabad, India
+bio: A passionate Web Developer and Competitive Programmer from India
+status: active
+badges:
+ - First Design
```

Comment:
remove these badges, there no badges for profile with first design and sharing is caring. // @vishalvivekm

Reply:
```suggestion
```